### PR TITLE
Fix/state selection

### PR DESCRIPTION
**Pull request by darlenedms**
#### What is the purpose of this pull request?

Fix shipping step for Chile that wasn't able to change region (state) after community (city) selection.
https://app.clubhouse.io/vtex/story/27751/problema-com-sele%C3%A7%C3%A3o-de-state-city-para-chile

Also fix flaky test that was returning empty string or null depending on computer/Node version.

#### How should this be manually tested?

1. https://darlene--vtexgame1.myvtex.com/checkout/cart/add/?sku=312&qty=1&seller=1&sc=2
2. Select Chile
3. Select any region (state)
4. Select any community (city)
5. Change region (state) and it should empty community (city) field

#### Screenshots or example usage

*BEFORE*
bad

*AFTER*
good

#### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Fixed
+
+- State/city selection on Chile because postalCode was not required when it should
```
Comment (salesfelipe) [inline CHANGELOG.md]:
```suggestion
- State/city selection on Chile because postalCode was not required when it should.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
+- Flaky tests that would return empty string or null depending of computer/Node version
```
Comment (salesfelipe) [inline CHANGELOG.md]:
```suggestion
- Flaky tests that would return empty string or null depending of computer/Node version.
```
